include/ibamr/IBImplicitStaggeredHierarchyIntegrator.h

```diff
@@ -233,10 +351,25 @@ class IBImplicitStaggeredHierarchyIntegrator : public IBHierarchyIntegrator
      */
     PetscErrorCode lagrangianSchurApply(Vec x, Vec y);
 
+	/*
+	 * Eulerian data for storing u and p DOFs indexing.
+	 */
+	std::vector<std::vector<int> > d_num_dofs_per_proc;
+	int d_u_dof_index_idx, d_p_dof_index_idx;
+	SAMRAI::tbox::Pointer<SAMRAI::pdat::SideVariable<NDIM, int> > d_u_dof_index_var;
+	SAMRAI::tbox::Pointer<SAMRAI::pdat::CellVariable<NDIM, int> > d_p_dof_index_var;
+
+	// Solvers and associated vectors.
+	bool d_solve_for_position;
     SAMRAI::tbox::Pointer<StaggeredStokesSolver> d_stokes_solver;
     SAMRAI::tbox::Pointer<StaggeredStokesOperator> d_stokes_op;
+	SAMRAI::tbox::Pointer<StaggeredStokesIBLevelRelaxationFACOperator> d_fac_op;
```

Comment:
> On Apr 24, 2015, at 12:31 AM, Amneet Bhalla <notifications@github.com> wrote:
> 
> In include/ibamr/IBImplicitStaggeredHierarchyIntegrator.h <https://github.com/IBAMR/IBAMR/pull/6#discussion_r29023144>:
> 
> >      SAMRAI::tbox::Pointer<StaggeredStokesSolver> d_stokes_solver;
> >      SAMRAI::tbox::Pointer<StaggeredStokesOperator> d_stokes_op;
> > +	SAMRAI::tbox::Pointer<StaggeredStokesIBLevelRelaxationFACOperator> d_fac_op;
> You mean within NoOpStokesSolver that I am creating? That seems fine but I just wanted to avoid cumbersome referencing e.g, d_stokes_solver->d_fac_op etc
> 
Just get this stuff from the Stokes solver unless there is a possibility that the FAC operator will be different than whatever is in the Stokes solver. E.g. it would be useful to test this out with different solvers, not just FAC using StaggeredStokesIBLevelRelaxationFACOperator.